templates/components/llamaindex/typescript/streaming/service.ts

```diff
@@ -1,3 +1,9 @@
+import {
```

Comment:
this file only covers LlamaCloud concerns (which is good).

But it shouldn't be there if we're not using LlamaCloud. I see two options:

1. we move it to `vectordbs/typescript/llamacloud` (like in the python code) - maybe we can do so by using a dynamic import of it?
2. we move `LLamaCloudFileService` to LITS (same package as LlamaCloudIndex)

what do you think? i kind of like option 2.